production/kubernetes/agent-bare.yaml.t

```diff
@@ -0,0 +1,115 @@
+apiVersion: v1
```

Comment:
The files in production/kubernetes/*.yaml are generated from Jsonnet, so we shouldn't template these, but rather the Jsonnet that they're generated from :)